tests/fixtures/policies/scripted_policy.py

```diff
@@ -0,0 +1,36 @@
+class ScriptedPolicy:
```

Comment:
can you add this class to `garage.np.policies` instead? it would be useful outside of tests.

Reply:
added